mcache/pyproject.toml

```diff
@@ -36,7 +36,7 @@ dynamic = [
 
 [project.optional-dependencies]
 deps = [
-    "python-binary-memcached==0.31.2; sys_platform != 'win32'",
+    "python-binary-memcached==0.31.4; sys_platform != 'win32'",
```

Comment:
This patch version includes https://github.com/jaysonsantos/python-binary-memcached/pull/258